debug/proto/debug.proto

```diff
@@ -28,3 +27,25 @@ message StatsResponse {
 	// total gc in nanoseconds
 	uint64 gc = 5;
 }
+
+// LogRequest queries service for logs
+message LogRequest {
+        // count is the count of events
+        int64 count = 1;
+        // relative time in seconds
+        // before the current time
+        // from which to show logs
+        int64 since = 2;
+        // stream logs continuously
+        bool stream = 3;
+}
+
+// Log is service log record
+message Log {
```

Comment:
Should this be Record? Or does it just look better as Log

Reply:
I think it makes sense to "map" the `proto` messages to `Go` code, so let's rename.